Cargo.toml

```diff
@@ -49,3 +49,9 @@ enumset = []
 embedded-svc = [ "dep:enumset", "dep:embedded-svc", "utils" ]
 wifi = []
 ble = []
+
+# currently published versions don't contain all relevant adjustments - using git dependencies for now
+[patch.crates-io]
+esp32-hal = { git = "https://github.com/esp-rs/esp-hal", package = "esp32-hal", rev = "da3ec47b30a5f598e904ffac0e10f94716cb4023" }
+esp32c3-hal = { git = "https://github.com/esp-rs/esp-hal", package = "esp32c3-hal", rev = "da3ec47b30a5f598e904ffac0e10f94716cb4023" }
+esp-hal-common = { git = "https://github.com/esp-rs/esp-hal", package = "esp-hal-common", rev = "da3ec47b30a5f598e904ffac0e10f94716cb4023" }
```

Comment:
Is `esp-hal-common` required? I ran ripgrep on the repo and didn't find any references to it. Not a big deal of course, just don't think it's necessary.

Reply:
Mhhh - last time it pulled `esp-hal-common` from the wrong source somehow. This time it apparently works w/o it. I can remove it if you like

Reply:
Your call, as I said not a big deal but I just would personally like to avoid pulling it in as a standalone package. I guess until we publish this though it's not really a big deal.

Reply:
Sorry I don't know what I was thinking, but those are clearly patches and not dependencies 🤦🏻